lib/kaffe/producer.ex

```diff
@@ -21,6 +21,25 @@ defmodule Kaffe.Producer do
 
   @kafka Application.get_env(:kaffe, :kafka_mod, :brod)
 
+  @typedoc """
+  A Kafka message can be represented as a tuple containing a key value pair of type binary()
+  """
+  @type message :: {key :: binary(), value :: binary()}
+
+  @typedoc """
+  A Kafka message can also be represented as a map, containing a `:key`, `:value` and `:headers`
```

Comment:
Small grammar recommendation, what do you think of this?

```
A Kafka message can also be represented as a map, containing `:key`, `:value`, and `:headers`
```

Reply:
Fixed